### PR TITLE
fix: consider null mask in sqlness display util

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Our proto is compact encoding, which omits the null values from the data vector. The original implement didn't consider this situation, and will produce incorrect results when converting a nullable column.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Comment by waynexia** [comment]
Looks like I update the lock file accidentally...

File: tests/runner/src/util.rs [hunk 1]
```diff
@@ -12,112 +12,107 @@
...
 use client::api::v1::column::Values;
 use client::api::v1::ColumnDataType;
+use common_base::BitVec;
 use tokio::io::AsyncWriteExt;
 use tokio::net::TcpSocket;
 use tokio::time;
 
 /// Check port every 0.1 second.
 const PORT_CHECK_INTERVAL: Duration = Duration::from_millis(100);
+const NULL_DATA_PLACEHOLDER: &str = "__NULL__";
```
Comment (evenyag) [inline tests/runner/src/util.rs:29]: Could we use `NULL` as what SQL displays?

Comment (evenyag) [reply tests/runner/src/util.rs:29]: > Could we use `NULL` as what SQL displays?

I think this isn't a critical problem, so I'd also approve this PR.

**Review by evenyag — APPROVED** [review]

File: tests/runner/src/util.rs (hunk 1 above)
Comment (killme2008) [reply tests/runner/src/util.rs:29]: Using `NULL` is better.

**Review by killme2008 — APPROVED** [review]

**Pull request merged**